2차/윤형진_정수-제곱근-판별.py

```diff
@@ -0,0 +1,12 @@
+import math
+
+def solution(n):
+    answer = 0
+    k = math.sqrt(n)
+    k = str(k)
+
+    if len(k[k.find("."):]) == 2 and int(k[k.find(".") + 1]) == 0 :
+        answer = (int(float(k)) + 1) * (int(float(k)) + 1)
+    else :
+        answer = -1
+    return answer
```

Comment:
이 부분도 구조 분해를 하면 좋을 것 같아요
```suggestion
    return (int(float(k)) + 1) * (int(float(k)) + 1) if len(k[k.find("."):]) == 2 and int(k[k.find(".") + 1]) == 0 else -1
```

이 부분은 if -return 형태로 하면 좋지만, 보통 마지막 값이 부정적인 의미보다는 긍정적인 의미를 사용하는 것이 좋습니다.